scene-lib/src/scenelib/annotations/Annotations.java

```diff
@@ -106,7 +106,8 @@ public static Annotation getRetentionPolicyMetaAnnotation(RetentionPolicy rp) {
         // meta-annotated by itself, we have to define the annotation
         // before we can create the annotation on it.
         aftRetentionPolicy = new EnumAFT("java.lang.annotation.RetentionPolicy");
-        adRetention = new AnnotationDef("java.lang.annotation.Retention");
+        adRetention = new AnnotationDef("java.lang.annotation.Retention",
+                                        "Special-case in scenelib/annotations/Annotations");
```

Comment:
Including the annotation name, e.g. the `java.lang.annotation.Retention` in the source info might be useful for debugging. At the moment all special cases have the same source.